src/core/storage/index.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+// $FlowIgnore[missing-export]
+import {TextEncoder, TextDecoder} from "util";
+
+/**
+ * A byte string represented as a Uint8Array. Each item in a byteString array
+ * is a Uint8 representing a byte value.
+ */
+export type ByteString = Uint8Array;
```

Comment:
Now that a `ByteString` is just a `Uint8Array` rather than a type with
side conditions, I think that we can just drop this type alias entirely.
What do you think?